src/common/styles/ButtonReset.ts

```diff
@@ -0,0 +1,11 @@
+import { css } from 'styled-components';
```

Comment:
We are pulling in tailwind in this branch https://github.com/holaplex/holaplex/pull/181

Are you using Styled components many places? Would you like to switch it over to TW now or just refactor it later?

Reply:
heheh... A bit late tho. Let's do it later because I made a ton of components using styled.